services/web/server/src/simcore_service_webserver/projects/_crud_api_read.py

```diff
@@ -64,14 +64,20 @@ async def list_projects(  # pylint: disable=too-many-arguments
     request: web.Request,
     user_id: UserID,
     product_name: str,
+    *,
```

Comment:
few questions here:
1. why if this is an api method is the `request` still there? it should be the app instead, there is no concept of request in an api. or is there?
2. the search query, where does it come from? in api/specs/web-server/_projects_crud.py I did not see it

Reply:
yes, i agree, it is inconsistent. The `projects` plugin carries many legacy inconsistencies that mainly @matusdrobuliak66 and me are trying to fix. 

WIll follow up in separate PR with it https://github.com/ITISFoundation/osparc-simcore/issues/6595